Altis_Life.Altis/core/functions/fn_loadDeadGear.sqf

```diff
@@ -6,5 +6,5 @@
     Description:
     BLAH
 */
-private _loadout = [_this,0,[[],[],[],[],[],[],"","",[],[]],[[]]] call BIS_fnc_param;
+params [["_loadout",[[],[],[],[],[],[],"","",[],[]],[[]]]];
```

Comment:
You could make a check here as well to check if ``param 0`` is not passed.

``if !(params ..) exitWith {}``